src/content/en/tools/chrome-user-experience-report/changelog.md

```diff
@@ -17,6 +17,19 @@ Tuesday of January 2020 after the data collection period has ended.
 In the list below, we've curated some release notes for each monthly dataset. Subscribe to our [CrUX Announce](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report-announce) mailing list or follow 
 [@ChromeUXReport](https://twitter.com/ChromeUXReport) on Twitter for release announcements.
 
+### 202002
+
+<dl>
+	<dt>Publication date</dt>
```

Comment:
Rick, can/should we link to the detailed note you publish on the forum? E.g. announcement of new experimental dataset, etc.